rosdep/python.yaml

```diff
@@ -5249,6 +5249,7 @@ python-transitions:
       pip:
         packages: [transitions]
   ubuntu:
+    focal: [python3-transitions]
```

Comment:
you need to add a key named `python3-transitions` to `python.yaml`, instead of modifying the one for python2.

Reply:
ok , I will create a new pull request for python3-transitions